frontends/android/BitBoxApp/app/src/main/java/ch/shiftcrypto/bitboxapp/Util.java

```diff
@@ -13,4 +14,8 @@ public static void systemOpen(Application application, String url) throws Except
             application.startActivity(intent);
         }
     }
+
+    public static void log(String msg) {
+        Log.d("ch.shiftcrypto.bitboxapp", "XYZ: " + msg);
```

Comment:
since you're here, remove the funny xyz? not sure what it's for.

Reply:
Done